crates/errors/src/lib.rs

```diff
@@ -35,8 +35,8 @@ mod helper;
 // META are meta related errors.
 
 declare_restate_error_codes!(
-    RT0001, RT0002, RT0003, RT0004, RT0005, RT0006, RT0007, RT0008, RT0009, META0001, META0002,
-    META0003, META0004, META0005, META0006, META0007, META0008, META0009, META0010, META0011,
+    RT0001, RT0002, RT0003, RT0004, RT0005, RT0006, RT0007, RT0008, RT0009, META0003, META0004,
+    META0005, META0006, META0009, META0010, META0011,
```

Comment:
What are we doing with retired error codes? Will they be reserved and cannot be used for future errors?